conf/env/PRESTASHOP.env

```diff
@@ -16,6 +16,7 @@ PS_FOLDER_INSTALL=installOLD
 ADMIN_MAIL=demo@reverb.com
 ADMIN_PASSWD=123456
 PS_HANDLE_DYNAMIC_DOMAIN=0
+PS_ERASE_DB=1
```

Comment:
I don't think I would want to reinstall the db every time that I start the container. Setting up test data over and over again is not a fun task.